packages/wrangler/src/dev/miniflare.ts

```diff
@@ -879,6 +920,9 @@ export async function buildMiniflareOptions(
 					proxy: true,
 				})),
 			},
+			...(config.experimentalAssets
```

Comment:
technically, for assets-only Workers, we should create a miniflare instance with just the AssetServerWorker here. Removing the entry Worker however broke things, and unfortunately I was kinds clueless as to why. BUT...I might have had a breakthrough in the meanwhile, I just need a bit more time to verify my assumptions & polish the code. I would like to ask that we don't block this PR on that 🙏

Reply:
Yup, fine by me. I think we'll eventually get to a place where we're _always_ at least attaching a Router Worker when assets are present, so don't spend too much time getting this running with ASW2 alone.

Reply:
indeed, that's kind of the architecture diagram I have in my head as well. The same principle will apply tho, since we'll need to figure out how to make the RW the entry point in the "miniflare pipeline". As it stands right now, that entry point seems to always be the entry-point Worker (which basically wraps the User Worker), so I'll need to have that figured out either way. But again, this can be done as a separate task